File: aula01-interop-dotnet/Point.cpp
using namespace System;

public ref class Point {
    protected:
        int _x, _y;
    public:
        Point(int x, int y) {
            this->_x = x;
            this->_y = y;
        }

        double getModule() {	 
            return Math::Sqrt((double)_x*_x + _y*_y);
        }
};

File: aula03-peverifiy-e-verifier/Point.cs
using System;


public class Point {
    public int x, y;
    public Point(int x, int y) {
        this.x = x;
        this.y = y;
    }
    public void Print() {
        Console.WriteLine("Point V1: ({0}, {1})", x, y);
    }
}